### ISSUE TITLE
Accessibility Audit: Keyboard focus indicator is not visible in high contrast mode on filter modal

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#931]
In a Macbook, turn on Increase Contrast (System Preferences -> Accessibility -> Display -> 'Increase Contrast Mode'). Then go to your Firefox browser and go to Settings -> General -> Language & Appearance -> Colors, and then click the 'Only with High Contrast themes' option on 'Override the colors specified by the page with your selections above'. 

On a Windows machine, in Firefox, go to Settings -> Vision -> Use High Contrast and set 'Turn on high contrast' to On. 

Now go to Firefox and look at the Filter modal. When you try to use the tab key to navigate between the different elements on the filter, there is no visual indicator as to which element you're currently focused on. The keyboard focus indicator should be highlighting the borders of the element that it's currently focused on.

NOTE: This is not replicable in Chrome.
